src/test/java/com/google/firebase/auth/FirebaseAuthIT.java

```diff
@@ -590,6 +652,52 @@ public void testGenerateSignInWithEmailLink() throws Exception {
     }
   }
 
+  private UserRecord createUser(
+      String uid,
+      @Nullable String phoneNumber,
+      @Nullable String email) throws Exception {
+    RandomUser randomUser = RandomUser.create();
```

Comment:
Unused variable. rm?

(Rabbit-hole:  I would've figured that javac would warn about this; apparently not. But before I figured that out, I noticed that we don't have warnings enabled in this repo. Enabling them yields 116 warnings messages. Yikes. I haven't looked at them, so have no idea if any are actually interesting or not.)